core/domain/exp_services_test.py

```diff
@@ -944,24 +1074,48 @@ def setUp(self):
             'param_changes': []
         }
 
-    def test_update_state_name(self):
-        """Test updating of state name."""
+    def test_add_state_cmd(self):
+        """ Test adding of states."""
+
         exploration = exp_services.get_exploration_by_id(self.EXP_ID)
+
+        self.assertNotIn('new state', exploration.states)
+
         exp_services.update_exploration(self.owner_id, self.EXP_ID, [{
-            'cmd': 'rename_state',
+            'cmd': exp_domain.CMD_ADD_STATE,
+            'state_name': 'new state',
+        }], 'Add state name')
+
+        exploration = exp_services.get_exploration_by_id(self.EXP_ID)
+        self.assertIn('new state', exploration.states)
+
+    def test_rename_state_cmd(self):
+        """Test updating of state name"""
+
+        exploration = exp_services.get_exploration_by_id(self.EXP_ID)
+
+        self.assertIn(feconf.DEFAULT_INIT_STATE_NAME, exploration.states)
+
+        exp_services.update_exploration(self.owner_id, self.EXP_ID, [{
+            'cmd': exp_domain.CMD_RENAME_STATE,
             'old_state_name': feconf.DEFAULT_INIT_STATE_NAME,
-            'new_state_name': 'new name',
+            'new_state_name': 'state',
         }], 'Change state name')
 
         exploration = exp_services.get_exploration_by_id(self.EXP_ID)
-        self.assertIn('new name', exploration.states)
+        self.assertIn('state', exploration.states)
         self.assertNotIn(feconf.DEFAULT_INIT_STATE_NAME, exploration.states)
 
     def test_update_state_name_with_unicode(self):
```

Comment:
Can we call this test_rename_state_cmd_with_unicode, for parity with the previous method?